l10n_it_delivery_note_base/__manifest__.py

```diff
@@ -31,4 +31,5 @@
         "views/stock_picking_transport_method.xml",
         "views/stock_picking_transport_reason.xml",
     ],
+    "post_init_hook": "create_dn_types_init_hook",
```

Comment:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#16installation-hooks

Gli hook vanno inseriti in un file `hooks.py`, e vanno chiamati esattamente `post_init_hook` etc.

Reply:
Quando contribuisci ad Odoo, segui le loro linee guida, quando contribuisci a OCA, segui le linee guida OCA, che in genere sono più restrittive.